### PR TITLE
MERC-6190: Remove bid/ask fields from Mercury v4 schema

### PR DESCRIPTION
**Pull request by martin-cll**
Requires:
- https://github.com/smartcontractkit/chainlink-common/pull/736

Supports:
- https://github.com/smartcontractkit/chainlink/pull/14252

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -63,17 +65,20 @@ require (
...
+	golang.org/x/text v0.17.0 // indirect
+	google.golang.org/genproto/googleapis/api v0.0.0-20240822170219-fc7c04adadcd // indirect
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20240822170219-fc7c04adadcd // indirect
 	google.golang.org/grpc v1.65.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
 replace (
+	// until merged upstream: https://github.com/omissis/go-jsonschema/pull/264
+	github.com/atombender/go-jsonschema => github.com/nolag/go-jsonschema v0.16.0-rtinianov
```
Comment (samsondav) [inline go.mod:80]: What is this?

File: go.mod [hunk 2]
```diff
@@ -1,14 +1,16 @@
 module github.com/smartcontractkit/chainlink-data-streams
 
-go 1.21
+go 1.22
+
+toolchain go1.22.6
```
Comment (samsondav) [inline go.mod:5]: Can we avoid bumping versions in this PR?

**Review by samsondav — DISMISSED** [review]

File: go.mod (hunk 2 above)
Comment (martin-cll) [reply go.mod]: I kept the go and toolchain versions the same as [`chainlink-common`](https://github.com/smartcontractkit/chainlink-common/blob/main/go.mod#L5).

File: go.mod (hunk 1 above)
Comment (martin-cll) [reply go.mod]: This is just to match common's go.mod file: https://github.com/smartcontractkit/chainlink-common/blob/main/go.mod#L111

**Review by brunotm — APPROVED** [review]

**Pull request merged**